### PR TITLE
Add seanet laplacian

### PR DESCRIPTION
**Pull request by mosem**

### CODE REVIEW

File: denoiser/batch_solvers/adversarial_bs.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+        disc_optimizer = torch.optim.Adam(discriminator.parameters(), lr=self.args.lr, betas=(0.9, self.args.beta2))
+        self._optimizers.update(({'discriminator_optimizer': disc_optimizer}))
+        self._losses_names += ['discriminator']
+
+
+    def run(self, data, cross_valid=False):
+        noisy, clean = data
+
+        generator = self._models['generator']
```
Comment (Or-Tal) [inline denoiser/batch_solvers/adversarial_bs.py:23]: use magic numbers instead of explicit strings

File: denoiser/models/modules.py [hunk 2]
```diff
@@ -422,4 +429,128 @@ def forward(self, y, y_hat):
...
+                results.append(disc(x))
+            elif i == self.num_D - 1:  # insert 0 - total frequency range / 2^num_D
+                downsampled_x = downsample2(x)
+                results.append(disc(downsampled_x))
+            else:  # insert total_frequency_range / 2^i - total_frequency_range/2^(i-1)
+                downsampled_x = downsample2(x)
+                laplacian = x - upsample2(downsampled_x)
+                results.append(disc(laplacian))
+                x = downsampled_x
+        return results
```
Comment (Or-Tal) [inline denoiser/models/modules.py:556]: long file, should be split to multiple files under modules package

File: launch_valentini.sh [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 python train.py \
   dset=valentini \
-  experiment=caunet_1 \
+  experiment=seanet_1 \
```
Comment (Or-Tal) [inline launch_valentini.sh:11]: please have a separate .sh file, leave the example file on demucs

Comment (Or-Tal) [reply launch_valentini.sh:11]: and add the separate file to .gitignore

**Review by mosem — COMMENTED** [review]
After merge of regular seanet branch to master, this is a very small commit. Merging this to master...

**Pull request merged**